CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## UNRELEASED
 
-BREAKING_CHANGES:
+BREAKING CHANGES:
```

Comment:
I'm assuming the underscore wasn't intentional? (don't want to break any automation)

Reply:
@wilkermichael Do you know?